agent/consul/acl_replication.go

```diff
@@ -52,7 +52,10 @@ type aclTypeReplicator interface {
 
 	// LocalMeta allows for type-agnostic metadata from the sorted local state
 	// can be retrieved for the purposes of diffing.
-	LocalMeta(i int) (id string, modIndex uint64, hash []byte)
+	LocalMeta(i int) (id string, modIndex uint64, hash []byte, hashMismatch bool)
```

Comment:
It looks like the `modIndex` return value is not used. Since this is adding another return value it seems like a good opportunity to remove an unused one to keep the signature a little more manageable.